src/services/EthService/network/helpers.ts

```diff
@@ -7,51 +7,61 @@ import { ETHERSCAN_API_KEY, INFURA_API_KEY } from '@config';
 import { DPathFormat, Network, NetworkId, NodeOptions, NodeType } from '@types';
 
 // Network names accepted by ethers.EtherscanProvider
-type TValidEtherscanNetwork = 'homestead' | 'ropsten' | 'rinkeby' | 'kovan' | 'goerli';
+type TValidEthersNetworkish = 'homestead' | 'ropsten' | 'rinkeby' | 'kovan' | 'goerli' | number;
```

Comment:
What does `networkish` refer to?!

Reply:
> What does `networkish` refer to?!

When setting a network using ethers.js they have string config specified for the above networks. But in the instance where it doesn't have a config for the network name, you need to use a chainid. This input type is called [Networkish](https://docs.ethers.io/v5/api/providers/types/#providers-Networkish) for some reason.